### ISSUE TITLE
Why does the votingClassifier in sklearn.ensemble gives higher accuracy than each of the classifiers that make it? and should i depend on it?

### ISSUE THREAD

**Issue by huhuhang** [huhuhang/ReadList#164]
**标题**：Why does the votingClassifier in sklearn.ensemble gives higher accuracy than each of the classifiers that make it? and should i depend on it?

**字数**：292 字

**摘要**：I am running an ML classifier on my data. I used SVM, RF and KNN. I used GScv for each of them and then used votingclassifier.The accuracy i got in each classifier independently was low, but from the hard and soft vote of the voting classifier is much higher! Why is that?

**链接**：https://datascience.stackexchange.com/questions/39694/why-does-the-votingclassifier-in-sklearn-ensemble-gives-higher-accuracy-than-eac
